### ISSUE TITLE
Librelec support

### ISSUE THREAD

**Issue by miloit** [muammar/mkchromecast#79]
Is librelec or openelec supported?

**Comment by muammar** [comment]
No, it is not. I do not own a librelec device. But in principle, if there is a python module to interact with it, I could try to do something.

**Comment by miloit** [comment]
Which module do you need?

**Comment by polyzen** [comment]
Seems you could use this: https://wiki.libreelec.tv/index.php?title=Pulseaudio#Network_Audio_Receiving

[Miraclecast](https://github.com/albfan/miraclecast) might be worth a consideration. Doesn't look like Kodi can be used as a Chrome- nor Miracast receiver, but there's at least a [package](https://aur.archlinux.org/packages/miraclecast/) for Arch Linux that claims to work with Arm.

http://www.pocket-lint.com/news/133437-what-is-miracast-and-how-is-it-different-from-chromecast-or-airplay

**Comment by miloit** [comment]
But how can I send audio to the chromecast device?

**Comment by polyzen** [comment]
Oh, right.. boy am I exhausted. There's apparently a Docker add-on, which is probably the way to go.

**Comment by miloit** [comment]
Can you send me the link?

**Comment by miloit** [comment]
Any news?
